event_listener/README.md

```diff
@@ -0,0 +1,113 @@
+This can be tested as a standalone service as follows;
+
+
+Installation
+============
+
+OS requirements
```

Comment:
I would suggest to simply write into the README how to start the docker compose up.
Then we would not require users to install specific python and what not!

Reply:
Cool with me. Although, in this case the readme needs to go at the top level of the project.